publish.css

```diff
@@ -41,6 +41,25 @@ button#HH:hover:before {
 	padding: 20px;
 	z-index: 5;
 }
+body:not(.popover.is-loaded) button#HH:hover:before { margin-left: -2vh; }
```

Comment:
is this for #116?

Reply:
Ah shoot, yeah it is. Forgot commits are auto added to open pull requests 🤦‍♀️

Reply:
ah yes, just appeared, it is 👍

Reply:
no worries, even better that this is fixed too :)